src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -552,7 +565,7 @@ GenTree* Compiler::impBaseIntrinsic(NamedIntrinsic        intrinsic,
 
     var_types simdBaseType = TYP_UNKNOWN;
 
-    if (intrinsic != NI_X86Base_Pause)
+    if (intrinsic != NI_X86Base_Pause && intrinsic != NI_X86Serialize_Serialize)
```

Comment:
Is this bit needed since it looks like `serialize` should be going down `impSerializeIntrinsic` ?

Reply:
Yes, I've made the changes. Thanks.